### PR TITLE
feat(release): Adds indices for Release on date_added and status

### PR DESCRIPTION
**Pull request by ahmedetefy**
This PR:
- Adds `index_together`  for model Release on `(organization, date_added)` and `(organization, status)`

### CODE REVIEW

File: src/sentry/migrations/0200_release_indices.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    CREATE INDEX CONCURRENTLY IF NOT EXISTS "sentry_release_organization_id_build_code_f93815e5_idx" ON "sentry_release" ("organization_id", "build_code");
+                    """,
+                    reverse_sql="DROP INDEX CONCURRENTLY IF EXISTS sentry_release_organization_id_build_code_f93815e5_idx",
+                ),
+                migrations.RunSQL(
+                    """
+                    CREATE INDEX CONCURRENTLY IF NOT EXISTS "sentry_release_organization_id_build_number_e1646551_idx" ON "sentry_release" ("organization_id", "build_number");
+                    """,
+                    reverse_sql="DROP INDEX CONCURRENTLY IF EXISTS sentry_release_organization_id_build_number_e1646551_idx",
+                ),
```
Comment (wedamija) [inline src/sentry/migrations/0200_release_indices.py:48]: These already exist, no need to recreate them

Comment (ahmedetefy) [reply src/sentry/migrations/0200_release_indices.py]: got it, done!

**Review by wedamija — APPROVED** [review]
You can merge this now, it won't run on production since it's marked as `is_dangerous`. Then you can just make an on-call request with ops (details in #teams-ops channel subject) linking to this pr and containing the sql you want them to run

**Comment by ahmedetefy** [comment]
> You can merge this now, it won't run on production since it's marked as `is_dangerous`. Then you can just make an on-call request with ops (details in #teams-ops channel subject) linking to this pr and containing the sql you want them to run

Sounds good! thanks

**Pull request merged**